### ISSUE TITLE
Scoreboard hidden

### ISSUE THREAD

**Issue by haer0248** [Puharesource/TitleManager#245]
**Describe the bug**
When player join the server, scoreboard still hidden.

**config.yml**
```
scoreboard:
  enabled: true
  title: '&a%player_world%&7-&a%player_direction%'
  lines:
  - '&7TPS &e%server_tps_1_colored%'
  - '&7LOC &e%player_x%&f,&e%player_y%&f,&e%player_z%'
  - '&7PING &e%{ping}ms'
  - '&7PLAYER &e%{safe-online}&7/&c%server_max_players%'
  - '&7BIOME &e%player_biome%'
  - '&7HOME &a%essentials_homes_set%&7/&c%essentials_homes_max%'
```

**Server information:**
 - TitleManager version: [2.1.6]
 - Spigot version: [PaperSpigot 1.15.1]

**Comment by haer0248** [comment]
**Warning: Nashorn engine is planned to be removed from a future JDK release**
When plugin load or reload it will display in console.

**Comment by Puharesource** [comment]
This should be fixed with the upcoming version (2.1.7).
The nashorn change is out of my reach and won't affect the plugin too much.
